templates/dashboard.php

```diff
@@ -71,25 +78,38 @@
                   <table class="table table-hover">
                     <thead>
                       <tr>
-                        <th><?php echo _("Host name"); ?></th>
-                        <th><?php echo _("IP Address"); ?></th>
-                        <th><?php echo _("MAC Address"); ?></th>
+                        <?php if ($arrHostapdConf['BridgedEnable'] == 1) : ?>
+                          <th><?php echo _("MAC Address"); ?></th>
+                        <?php else : ?>
+                          <th><?php echo _("Host name"); ?></th>
+                          <th><?php echo _("IP Address"); ?></th>
+                          <th><?php echo _("MAC Address"); ?></th>
+                        <?php endif; ?>
                       </tr>
                     </thead>
                     <tbody>
+                        <?php if ($arrHostapdConf['BridgedEnable'] == 1) : ?>
+                          <tr>
+                            <td><?php echo _("<em>Bridged AP mode is enabled. For Hostname and IP, see your router's admin page.</em>");?></td>
```

Comment:
```suggestion
                            <td><small class="text-muted"><?php echo _("Bridged AP mode is enabled. For Hostname and IP, see your router's admin page.");?></small></td>
```

Good call, I should familiarize myself more with bootstrap